### PR TITLE
[Bug] Add default og tags

### LINKED ISSUE #5872: 🐛 Meta tags not rendered for preview
## 🐛 Bug

Meta tags are not being be rendered for preview.

## 🦋 Expected Behaviour

Meta tags should be rendered for preview.

## 📋 Steps to Reproduce

1. Scrape **https://talent.canada.ca/en/** with an OpenGraph tool (ex. https://www.opengraph.xyz/)
2. Observe the description has no value

## ✅ Acceptance Criteria

- [ ] meta tags should be rendered for preview

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #5872 

## 👋 Introduction

Adds fallback, default open graph tags to `index.html`

## 🕵️ Details

This only adds English versions to remain under the soft 155 character limit for SEO. Should we tweak the text to shorten in and include French?

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

> **Note**
> Hard to test because `react-helmet-async` takes over and replaces the tags.

1. Comment out the `<SEO />` component in `HomePage.tsx` and `Layout.tsx`
2. Build the app `npm run build`
3. Navigate to the homepage `/en`
4. Confirm the open graph tags exist

### CODE REVIEW

File: apps/web/public/index.html [hunk 1]
```diff
@@ -4,7 +4,21 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
   <title><%= htmlWebpackPlugin.options.title %></title>
+  <meta name="description" content="GC Digital Talent is the new recruitment platform for digital and tech jobs in the Government of Canada. Apply now!">
+  <meta property="og:url" content="https://talent.canada.ca/en/">
```
Comment (mnigh) [inline apps/web/public/index.html:10]: Unless we can control the locale, best to remove it?

```suggestion
  <meta property="og:url" content="https://talent.canada.ca/">
```

**Review by mnigh — COMMENTED** [review]
> This only adds English versions to remain under the soft 155 character limit for SEO. Should we tweak the text to shorten in and include French?

Yes, unless there is a way to determine the locale and show the corresponding string via webpack or something, I would vote for including English and French versions by default.

**Comment by esizer** [comment]
> Yes, unless there is a way to determine the locale and show the corresponding string via webpack or something, I would vote for including English and French versions by default.

Add localization in [localize og tags](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6015/commits/d234bbf9de4c4bf6cf434cd9865f83fda9482cb0)

**Comment by petertgiles** [comment]
Can probably close #2408, too.

File: packages/webpack-config/webpack.base.js [hunk 2]
```diff
@@ -9,6 +9,15 @@ require('dotenv').config({ path: './.env' });
 const shell = require("shelljs");
 const fs = require("fs");
 
+const meta = {
+  title: "GC Digital Talent | Talent numérique du GC",
+  description: "Recruitment platform for digital jobs in the Government of Canada. Recrutement pour les emplois numériques au gouvernement du Canada.",
```
Comment (mnigh) [inline packages/webpack-config/webpack.base.js:14]: The French version of _Recruitment platform_ does not include the word platform.

**Review by mnigh — COMMENTED** [review]
Like you said, hard to test put it appears to be working. My only concern is around the copy in the description.

## Screenshot
Screen Shot 2023-03-22 at 10 46 02

File: packages/webpack-config/webpack.base.js (hunk 2 above)
Comment (esizer) [reply packages/webpack-config/webpack.base.js:14]: Oops, that is an important piece

**Review by mnigh — APPROVED** [review]
LGTM

**Pull request merged**